charts/prometheus/templates/server/sts.yaml

```diff
@@ -110,21 +110,32 @@ spec:
           {{- if .Values.server.baseURL }}
             - --web.external-url={{ .Values.server.baseURL }}
           {{- end }}
+          {{- if .Values.server.listen.host }}
+            - --web.listen-address={{ .Values.server.listen.host }}:9090
```

Comment:
i guess this will not match with "listen: {}" from the values.

Reply:
What do you mean? I initialized this "listen" block with an empty map "{}", because otherwise I get this error when checking `if .Values.server.listen.host`
>  <.Values.server.listen.host>: nil pointer evaluating interface {}.host

The alternative is to do `if .Values.server.listen and .Values.server.listen.host`, both in deploy.yaml and sts.yaml.